Assets/Scripts/CreatureController.cs

```diff
@@ -1,14 +1,18 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
-public class CharacterController : MonoBehaviour {
-    // Raw hp amount, the player could have anywhere from 1-X
-    float hp;
-    public string characterName;
+public class CreatureController : MonoBehaviour {
     public HealthbarController healthbarController;
 
-    // Start is called before the first frame update
+    public string characterName;
+
+    public CombatManager combatManager;
+
+    public GameObject targetingArrow;
+
+    bool isSelected;
+
+    // Raw hp amount, the player could have anywhere from 1-X
+    float hp;
```

Comment:
Should we keep a consistent convention with variable orders?  I was under the impressions we'd want to go private then public variables.

Reply:
Oh I thought public then private.

Reply:
Oh!